src/pages/home/home.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { ReactElement } from 'react';
 
-const Home = (): ReactElement => <h1>Home page</h1>;
+import NavigationBar from '../../components/navigation-bar/navigation-bar';
+
+const Home = (): ReactElement => <NavigationBar />;
```

Comment:
Home should be navigation with main content on righ, as in screen
<img width="1782" alt="Screenshot 2021-12-21 at 22 29 14" src="https://user-images.githubusercontent.com/91533721/146993493-f50bf7bb-63a9-43df-9e1d-489dd742c0a6.png">